CMakeLists.txt

```diff
@@ -462,7 +462,7 @@ configure_file(common/SC_Version.hpp.in common/SC_Version.hpp)
 include_directories(${CMAKE_CURRENT_BINARY_DIR}/common)
 
 file(GLOB_RECURSE ClassLibrary SCClassLibrary/*sc)
-file(GLOB_RECURSE HelpSource   HelpSource/*)
+file(GLOB_RECURSE HelpSource   HelpSource/**)
```

Comment:
Could just explain what this does and why it's needed?

Reply:
I remember having problems that some files were not copied, but this seems this was caused when doing some experiments with MathJax. Normally in glob, `*` matches everything except `/` and `**` anything, so including all sub directories.

I just ran a build with the old CMakeLists from a clean build dir and it still works, so I will rebase/edit the commit.